make/modules/java.base/Lib.gmk

```diff
@@ -213,9 +213,8 @@ $(eval $(call SetupJdkLibrary, BUILD_SYSLOOKUPLIB, \
     LDFLAGS := $(LDFLAGS_JDKLIB), \
     LDFLAGS_linux := -Wl$(COMMA)--no-as-needed, \
     LDFLAGS_aix := -brtl -bexpfull, \
-    LIBS := $(LIBCXX), \
-    LIBS_linux := -lc -lm -ldl, \
-    LIBS_aix := -lc -lm -ldecNumber, \
+    LIBS_linux := -lm -ldl, \
+    LIBS_aix := -lm -ldecNumber, \
```

Comment:
This lib doesn't set LINK_TYPE. Did it add LIBCXX without needing it?

Reply:
Apparently. It is a minimal library, consisting of a single, (almost) empty file. It also set `-lc` which is not needed. My guess is that this is the result of copy-paste code.